packages/core/src/V2/Node.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable, Subscription, SubscriptionLike } from 'rxjs';
+
+export interface INodePorts {
```

Comment:
Where useful, please provide a little doc bloc with the purpose of interface/type